rust/private/rust.bzl

```diff
@@ -328,6 +328,10 @@ def _rust_library_common(ctx, crate_type):
     deps = transform_deps(ctx.attr.deps)
     proc_macro_deps = transform_deps(ctx.attr.proc_macro_deps + get_import_macro_deps(ctx))
 
+    build_script = ctx.attr.build_script
+    if build_script != None:
+        build_script = transform_deps([ctx.attr.build_script])[0]
```

Comment:
I think we can create the DepVariantInfo here, given that we know that build_script contains a BuildInfo, so we don't need to call transform_deps. Same below.